metecho/api/models.py

```diff
@@ -745,6 +769,26 @@ def root_project(self) -> Project:
             return self.epic.project
         return self.project
 
+    @property
+    def branch_url(self) -> Optional[str]:
+        if self.branch_name:
+            return f"{self.root_project.repo_url}/tree/{self.branch_name}"
+        return None
+
+    @property
+    def branch_diff_url(self) -> Optional[str]:
+        base_branch = self.get_base()
+        branch = self.branch_name
+        if base_branch and branch:
+            return f"{self.root_project.repo_url}/compare/{base_branch}...{branch}"
+        return None
+
+    @property
+    def pr_url(self) -> Optional[str]:
+        if self.pr_number:
+            return f"{self.root_project.repo_url}/pull/{self.pr_number}"
+        return None
```

Comment:
These properties were previously defined in the serializer but we need them to log Task activity so they were moved to the model